packages/shell-chrome/src/utils.js

```diff
@@ -87,24 +87,52 @@ export function convertInputDataToType(inputType, value) {
     }
 }
 
+function getAttributeValue(value, type) {
```

Comment:
getDisplayName?

Let's leave this for another PR, we use "attribute" everywhere so makes sense to keep it consistent for now